app/controllers/transactions_controller.rb

```diff
@@ -1,18 +1,18 @@
 class TransactionsController < ApplicationController
   def index
-    @category = Category.find(params[:category_id])
+    @category = Category.includes(:transactions).find(params[:category_id])
     @transactions = @category.transactions.order(created_at: :desc)
     @total_amount = @category.total_amount
   end
 
   def new
     @category = Category.find(params[:category_id])
-    @transaction = @category.transactions.build
+    @transaction = Transaction.new
   end
 
   def create
     @category = Category.find(params[:category_id])
-    @transaction = @category.transactions.build(transaction_params.merge(author_id: current_user.id))
+    @transaction = Transaction.new(transaction_params.merge(author_id: current_user.id))
 
     if @transaction.save
       @transaction.transaction_categories.create(category_id: @category.id)
```

Comment:
- [ ] Please re-investigate the transactions create method in order to resolve this issue. 🙏  
![Action Controller_ Exception caught - Google Chrome 11_23_2023 12_38_33 PM](https://github.com/hetrox8/Budget_App/assets/47625407/07fbb532-8d5b-4b8c-a0a9-9f06e626538c)

Reply:
well resolved! ✔️